src/js/components/messaging/subscribe.js

```diff
@@ -0,0 +1,52 @@
+import { API } from "../../api";
+
+export const subscribe = ($customCheckbox, $checkbox) => {
```

Comment:
this isn't so much a component as it is a library, right? perhaps it should be in `src/js/notifications/subscribe.js`?

Reply:
https://github.com/OpenUpSA/muni-portal-frontend/issues/70